aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -1126,6 +1127,88 @@ inline void NestedTensor_reshape_copy(
 }
 } // namespace
 
+  /**
+   * Create a buffer tensor that is a view of self
+   *
+   * This serves as the boundary between nested and non nested tensor
+   * view conversions
+   *
+   * @return Returns a new non nested tensor that
+   * aliases the same storage as self
+   */
+Tensor NestedTensor_to_buffer(const Tensor& self) {
+  TORCH_CHECK(self.is_nested(), "Can only create a buffer from Nested Tensor");
+  auto* nt_self = get_nested_tensor_impl(self);
+  return nt_self->get_buffer();
+}
+Tensor NestedTensor_to_buffer_copy(const Tensor& self) {
+  TORCH_CHECK(self.is_nested(), "Can only create a buffer from Nested Tensor");
+  auto* nt_self = get_nested_tensor_impl(self);
+  return nt_self->get_buffer().clone();
+}
+
+/**
+ * Create a nested tensor that is a view of a buffer
+ *
+ * This serves as the boundary between non nested tensor and nested
+ * view conversions
+ *
+ * @return Returns a nested tensor that
+ * aliases the same storage as buffer
+ */
+Tensor NestedTensor_from_buffer(
+    const Tensor& buffer,
+    const Tensor& reference_tensor) {
```

Comment:
I am a bit confused by this.
Why isn't this a regular factory function that takes buffer, size, stride, offset ?